docs/api/07_coupons/create-coupon.mdx

```diff
@@ -144,10 +167,13 @@ import TabItem from '@theme/TabItem';
 |--|--|--|
 | name | String &nbsp &nbsp &nbsp<Required>**Required**</Required> | Coupon name |
 | code | String &nbsp &nbsp &nbsp<Required>**Required**</Required> | Code identifying the coupon |
-| amount_cents | Integer &nbsp &nbsp &nbsp<Required>**Required**</Required> | Amount of the coupon |
-| amount_currency | String &nbsp &nbsp &nbsp<Required>**Required**</Required> | Currency of the amount |
+| amount_cents | Integer &nbsp &nbsp &nbsp<Optional>**Optional (required only for `fixed_amount` type)**</Optional> | Amount of the coupon |
+| amount_currency | String &nbsp &nbsp &nbsp<Optional>**Optional (required only for `fixed_amount` type)**</Optional> | Currency of the amount |
 | expiration | String &nbsp &nbsp &nbsp<Required>**Required**</Required> | Expiration type.<br></br>It can be `time_limit` or `no_expiration` |
-| expiration_duration | Integer &nbsp &nbsp &nbsp<Required>**Required (optional only for `no_expiration` type)**</Required> | Coupon duration. |
+| expiration_date | String &nbsp &nbsp &nbsp<Optional>**Optional (required only for `time_limit` type)**</Optional> | Coupon expiration date. |
+| frequency | String &nbsp &nbsp &nbsp<Required>**Required**</Required> | Frequency type.<br></br>It can be `once` or `recurring` |
+| frequency_duration | String &nbsp &nbsp &nbsp<Optional>**Optional (required only for `recurring` frequency type)**</Optional> | This field determines number of billing periods to which the coupon is applied. |
```

Comment:
This field determines the number of billing periods to which the coupon applies.